backend/stories/models.py

```diff
@@ -2,6 +2,7 @@
 from sqlalchemy import Column, ForeignKey, Integer, String, Table, JSON
 from sqlalchemy.orm import relationship
 from sqlalchemy.ext.declarative import declarative_base
+import json
```

Comment:
I'm pretty sure we are using the following convention when importing. I'll check it on our slack project channel though:

1. Standard library imports
2. Third party imports
3. Local application imports

Maybe more for future commits than these ones

Reply:
oh I hadn't notice about this one. Will change :)

Reply:
taking a closer look, `json` was already being imported in the first line 😅